shared/components/Speaker/index.scss

```diff
@@ -8,6 +8,12 @@
     font-weight: normal;
     &--bold {
       font-weight: bold;
+      text-decoration: underline;
+      margin-bottom: 5px;
+    }
+
+    &--bold.placeholder {
```

Comment:
This doesn't look very BEM-ish. Probably placeholder should be another element of name
i.e. `Speaker__name__placeholder`